.gitignore

```diff
@@ -7,4 +8,4 @@ BiblioPixel.egg*
 *.pyc
 .cache
 .coverage
-htmlcov
\ No newline at end of file
+htmlcov
```

Comment:
Gotcha... yes, my editor always adds the newline on save. I use Atom. (I know I know... console editor or die. But they can pry Atom from my cold dead hands)